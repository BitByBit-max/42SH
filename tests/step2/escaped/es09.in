echo \n\n \n \t \t \z \t\a
