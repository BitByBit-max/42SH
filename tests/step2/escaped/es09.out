nn n t t z ta
